### PR TITLE
7.22 initialise docs matrices with new version

### PR DESCRIPTION
**Pull request by psavidis**
Related-to: https://github.com/camunda/camunda-bpm-platform/issues/4273

### CODE REVIEW

File: content/user-guide/testing/_index.md [hunk 1]
```diff
@@ -411,6 +411,12 @@ With Camunda 7.17.0 the project was moved into the [Camunda 7 repository](https:
     <td>7.20.0<br/>7.21.0</td>
     <td>7.20.0<br/>7.21.0</td>
   </tr>
+  <tr>
+    <td>camunda-bpm-assert</td>
+    <td>3.25.3</td>
```
Comment (psavidis) [inline content/user-guide/testing/_index.md:416]: The version in the `camunda-bpm-assert-root` module is transitively brought by Spring Boot [version](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/parent/pom.xml#L24) and the corresponding dependency version is `3.25.3`.

💡 The following artifacts use explicitly `assertj 2.9.1` from the [parent](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/parent/pom.xml#L51)
- `camunda-commons` ([ref](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/commons/pom.xml#L41-L45))
- `camunda-connect` ([ref](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/connect/pom.xml#L57-L61))
- `camunda-template-engines-freemarker` ([ref](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/freemarker-template-engine/pom.xml#L38-L44))
- `camunda-core-internal-dependencies` ([ref](https://github.com/camunda/camunda-bpm-platform/blob/e5deaad2928a01da421419f1542de894a51eb5b6/internal-dependencies/pom.xml#L94-L97))

❓ Double check with the reviewer the assertj versions have nothing to do with the `assertj` used in the projects and verify the correctness of the committed entry

**Comment by psavidis** [comment]
❓ Verify with the reviewer if the changes need to be merged to [7.22](https://github.com/camunda/camunda-docs-manual/tree/7.22) branch as well which is already created without them.

File: content/user-guide/testing/_index.md (hunk 1 above)
Comment (danielkelemen) [reply content/user-guide/testing/_index.md:416]: 👍 That's correct, `3.25.3`.

**Review by danielkelemen — APPROVED** [review]
👍 Looks good!

**Pull request merged**